Release 2025.11.0 (#5796)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2025/index.md

#### Submit entitlement management integrations

Independent Software Vendors (ISVs) can now submit SCIM 2.0-based entitlement management integrations to the Okta Integration Network (OIN). This enhancement enables customers and IT admins to discover, manage, and assign fine-grained entitlements such as roles and permissions directly from Okta. By standardizing entitlement management, organizations can automate access assignments and streamline Identity Governance, ensuring users receive the right access and roles without manual intervention. For more information, see [Submit an integration with the OIN Wizard](https://developer.okta.com/docs/guides/submit-oin-app/scim/main/). <!-- OKTA-1025782 ENTITLEMENTS_SUBMISSION -->

#### Incognito session validation in OIN Submission Tester

To improve test reliability, the OIN Submission test flow now automatically detects conflicting incognito tabs or windows. If active incognito tabs or windows are found, the prompt allows you to close all detected incognito tabs or windows with a single action before the test begins. <!-- OKTA-943185 -->

#### New System Log event for AD agent changes

> A new System Log event (`system.agent.ad.config_change_detected`) reports when Okta Support modified an AD agent configuration.

Reviewer: ```suggestion
The System Log event (`system.agent.ad.config_change_detected`) reports when Okta Support modified an AD agent configuration.
```

#### New System Log event for AD agent changes

A new System Log event (`system.agent.ad.config.change.detected`) reports when Okta Sspport modified an AD agent configuration.

#### Developer documentation updates in 2025.11.0

* Removed API placeholder pages from the **References** section. Links to these legacy pages now automatically redirect to the [API docs](https://developer.okta.com/docs/api/).

* New [release notes for Okta Access Gateway APIs](/docs/release-notes/2025-okta-access-gateway/) are now available.

* The [Okta IDX SDK for Golang](https://github.com/okta/okta-idx-golang) and the [Golang sample apps](https://github.com/okta/samples-golang) were archived and the corresponding Go language documentation has been removed from the Identity Engine (IDX) guides. Additionally, the main SDK page is updated to include details about these archived SDKs. See [Archived SDKs](/code/#archived-sdks).

#### Bugs fixed in 2025.11.0

> * A new System Log event (`system.agent.ad.config.change.detected`) reports when Okta Support modified an AD agent configuration. (OKTA-1047077)

Reviewer: ```suggestion

```
Remove from bug section
